util/coverage/unit.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
I don't really understand what belongs in `unit.py`. Would it make sense to name it `compilation_unit.py`?

Reply:
Renamed this to `unittest_coverage.py` as we discussed.